### PR TITLE
[backport] crosssdk is droped in oe-core

### PR DESCRIPTION
**Pull request by quaresmajose**
fixes https://github.com/OE4T/meta-tegra/issues/1246

### CODE REVIEW

File: recipes-devtools/gcc-for-nvcc/gcc-8-crosssdk.inc [hunk 1]
```diff
@@ -8,5 +8,5 @@ SYSTEMLIBS1 = "${SDKPATHNATIVE}${libdir_nativesdk}/"
 
 GCCMULTILIB = "--disable-multilib"
 
-DEPENDS = "virtual/${TARGET_PREFIX}binutils-crosssdk gettext-native ${NATIVEDEPS}"
-PROVIDES = "virtual/${TARGET_PREFIX}cuda-gcc-crosssdk virtual/${TARGET_PREFIX}cuda-g++-crosssdk"
+DEPENDS = "virtual/${TARGET_PREFIX}binutils gettext-native ${NATIVEDEPS}"
+PROVIDES = "virtual/${TARGET_PREFIX}cuda-gcc-crosssdk virtual/${TARGET_PREFIX}cuda-g++"
```
Comment (madisongh) [inline recipes-devtools/gcc-for-nvcc/gcc-8-crosssdk.inc:12]: Shouldn't the `-crosssdk` suffix be dropped for both of the entries in this PROVIDES?

Comment (quaresmajose) [reply recipes-devtools/gcc-for-nvcc/gcc-8-crosssdk.inc:12]: Right, forgot this one as in https://git.yoctoproject.org/poky/diff/meta/recipes-devtools/gcc/gcc-crosssdk.inc?id=768c89e306d0486acf8bc76cecef90436f6551b3
Sorry I need to manually change the patch and when I see it building don't look any more.
I will update and fix.

**Review by ichergui — APPROVED** [review]
LGTM 
Thanks @quaresmajose

**Comment by madisongh** [comment]
Thanks!

**Pull request merged**